.travis.yml

```diff
@@ -70,15 +56,30 @@ matrix:
           sources:
             - ubuntu-toolchain-r-test # if we don't specify this, the libstdc++6 we get is the wrong version
           packages:
+            - lib32stdc++6 # https://github.com/flutter/flutter/issues/6207
```

Comment:
Please ignore the changes in this file, I've send out these changes in a separate PR (https://github.com/flutter/plugins/pull/145) and will rebase this PR on top of the other.